### DISCUSSION TITLE
Why is 'Philly' not going to be named 'Philly Nice' ?

### DISCUSSION THREAD

**Discussion by Lucky-56** [KadeArchive/Kade-Engine#771] [Q&A] [answered]
Is it because of compatability with FileDaddy or any other reason it got changed back?

**Comment by puyoxyz** [comment, marked as answer]
backwards comp with mods

**Reply by Lucky-56** [reply]
speaking of compatibilty why is 'dadbattle' renamed to 'dad-battle'

**Reply by theDetourist** [reply]
It's not, you only use dashes in the folder/file names so it doesn't interfere with old songs that don't have spaces. So spaces in the name of the songs will then be converted to dashes instead so both the name of the song and its folder and files match properly.

**Reply by Lucky-56** [reply]
I know that but for testing purposes of [#769](https://github.com/KadeDev/Kade-Engine/issues/769#issuecomment-856359001) 'Dad Battle' didn't work because the 'dadbattle' folder didn't exist and therefore not been replaced

**Reply by Lucky-56** [reply]
I'm working on a fix currently #789
